sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 14.1.0 (2021-09-01)
```

Comment:
This date needs to be within 30 days of the actual release date or the release pipeline will fail.